src/pages/basic_data/app_admin/CentralImportPage.tsx

```diff
@@ -1,34 +1,76 @@
 import { useTranslation } from 'react-i18next';
 import { useFetch } from '../../../utils/hooks/useFetch';
 import { Registration } from '../../../types/registration.types';
 import { SearchApiPath } from '../../../api/apiPaths';
 import { SearchResponse } from '../../../types/common.types';
 import { ListSkeleton } from '../../../components/ListSkeleton';
 import { CentralImportResultItem } from './CentralImportResultItem';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+import { SearchParam } from '../../../utils/searchHelpers';
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { stringIncludesMathJax, typesetMathJax } from '../../../utils/mathJaxHelpers';
 
 export const CentralImportPage = () => {
   const { t } = useTranslation('basicData');
+  const history = useHistory();
+  const params = new URLSearchParams(history.location.search);
+  const resultsParam = params.get(SearchParam.Results);
+  const fromParam = params.get(SearchParam.From);
+  const rowsPerPage = (resultsParam && +resultsParam) || ROWS_PER_PAGE_OPTIONS[1];
+  const page = (fromParam && resultsParam && Math.floor(+fromParam / rowsPerPage)) || 0;
 
-  const [results, isLoadingResults] = useFetch<SearchResponse<Registration>>({
-    url: SearchApiPath.Registrations,
+  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
+    url: `${SearchApiPath.Registrations}?${params.toString()}`,
     errorMessage: t('feedback:error.search'),
   });
-  const publications = results?.hits ?? [];
+
+  const updatePath = (from: string, results: string) => {
+    params.set(SearchParam.From, from);
+    params.set(SearchParam.Results, results);
+    history.push({ search: params.toString() });
+  };
+
+  const publications = searchResults?.hits ?? [];
+
+  useEffect(() => {
+    if (searchResults?.hits.some(({ entityDescription }) => stringIncludesMathJax(entityDescription?.mainTitle))) {
+      typesetMathJax();
+    }
+  }, [searchResults]);
+
   return (
     <>
       <Typography variant="h3" component="h2" paragraph>
         {t('publications')}
       </Typography>
-      {isLoadingResults ? (
+      {isLoadingSearchResults ? (
         <ListSkeleton minWidth={100} maxWidth={100} height={100} />
       ) : (
-        publications.length > 0 && (
-          <List>
-            {publications.map((publication) => (
-              <CentralImportResultItem publication={publication} key={publication.identifier} />
-            ))}
-          </List>
+        searchResults && (
+          <>
+            <Typography variant="subtitle1">{t('search:hits', { count: searchResults.size })}:</Typography>
+            <Divider />
+            <List>
+              {publications.map((publication, index) => (
+                <CentralImportResultItem publication={publication} key={publication.identifier + index} />
```

Comment:
kan samme registrering listes flere ganger? 🤔

Reply:
du har ørneblikk!  Jeg glemte å fjerne den etter å ha satt den på for mock-data. (siden mock-data har alle samme id, så ble det mye rusk i loggen)
fikset